templates/app-esm/app.js

```diff
@@ -21,7 +21,7 @@ export default async function (fastify, opts) {
     options: Object.assign({}, opts)
   })
 
-  // This loads all plugins defined in routes
+  // This loads all routes defined in routes
```

Comment:
The whole sentence does seem correct: a plugin contains the routes

Maybe we should say "this loads all the plugins defined in routes folder"